3rdparty/README.md

```diff
@@ -0,0 +1,6 @@
+# 3rdparty modules documentation
+
+## AdjustSdk.framework
+Source: https://github.com/adjust/ios_sdk/releases/download/v4.29.4/AdjustSdkDynamic.framework.zip
+Version: 4.29.4\
```

Comment:
extra slash here?

Reply:
This is to end the line because on my preview `Documentation` would be on the same line otherwise